signInAndSignOut_feature/UI/DashboardController/DashboardViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DashboardViewController.swift
+//  signInAndSignOut_feature
+//
+//  Created by SID on 21/08/2024.
+//
+
+import UIKit
+
+class DashboardViewController: UIViewController, DataSendingProtocol {
```

Comment:
Instead of Inheriting the DashboardViewController from DataSendingProtocol 
Use extension out side the body of class to inheriate the Dashbaord from DataSendingProtocl